x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/termsenum/action/TermsEnumRequest.java

```diff
@@ -21,12 +22,15 @@
 
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.Objects;
+
 
 /**
  * A request to gather terms for a given field matching a string prefix
  */
 public class TermsEnumRequest extends BroadcastRequest<TermsEnumRequest> implements ToXContentObject {
 
+    public static final IndicesOptions DEFAULT_INDICES_OPTIONS = SearchRequest.DEFAULT_INDICES_OPTIONS;
```

Comment:
Safe to assume terms_enum options will always be aligned with that of search?

Reply:
That's the intent of the PR, yes.